examples/olsconfig.yaml

```diff
@@ -23,7 +23,7 @@ llm_providers:
       - name: ibm/granite-13b-chat-v2
 ols_config:
   reference_content:
-#    product_docs_index_path: "./vector-db/ocp-product-docs"
+#    product_docs_index_path: "./vector_db/ocp_product_docs/4.15"
```

Comment:
@ilan-pinto's latest changes switched the index id to `ocp-product-docs-4_15`

so if you align around that index id, you'll avoid having to touch a bunch of places like this one:
https://github.com/openshift/lightspeed-service/blob/0a90423f897711fb58105f7e6343df2aee92e1c6/tests/config/cluster_install/ols_configmap.yaml#L22-L23

Reply:
https://github.com/openshift/lightspeed-service/pull/554 fixes the index ids for the remaining spots i'm aware of that "matter"